MobileWallet/Screens/AppEntry/Splash/SplashViewController.swift

```diff
@@ -226,100 +226,25 @@ class SplashViewController: UIViewController, UITextViewDelegate {
-                        } else {
-                            DispatchQueue.main.async { [weak self] in
-                                guard let _ = self else { return }
-                                onSuccess()
-                            }
-                        }
-                    } else {
-                        let reason = error?.localizedDescription ?? NSLocalizedString("Failed to authenticate", comment: "Failed Face/Touch ID alert")
-                        TariLogger.error("Biometrics auth failed", error: error)
-                        DispatchQueue.main.async { [weak self] in
-                            guard let self = self else { return }
-                            self.authenticationFailedAlertOptions(reason: reason, onSuccess: onSuccess)
-                        }
-                    }
-                }
-            }
-        case .none:
-            let alert = UIAlertController(title: NSLocalizedString("Authentication Error", comment: "No biometric or passcode") ,
-                                          message: NSLocalizedString("Tari Aurora was not able to authenticate you. Do you still want to proceed?", comment: "No biometric or passcode"),
-                                          preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: NSLocalizedString("Try again", comment: "Try again button"),
-                                          style: .cancel,
-                                          handler: { [weak self] _ in
-                                            self?.authenticateUser(onSuccess: onSuccess)
-            }))
-
-            alert.addAction(UIAlertAction(title: NSLocalizedString("Proceed", comment: "Proceed button"), style: .default, handler: { _ in
-                onSuccess()
-            }))
-
-            self.present(alert, animated: true, completion: nil)
-        }
-    }
-
-    private func authenticationFailedAlertOptions(reason: String, onSuccess: @escaping () -> Void) {
-        let alert = UIAlertController(title: NSLocalizedString("Authentication failed", comment: "Auth failed"), message: reason, preferredStyle: .alert)
-        alert.addAction(UIAlertAction(title: NSLocalizedString("Try again", comment: "Try again button"), style: .default, handler: { [weak self] _ in
-            guard let self = self else { return }
-            self.authenticateUser(onSuccess: onSuccess)
-        }))
-
-        alert.addAction(UIAlertAction(title: NSLocalizedString("Open settings", comment: "Open settings button"), style: .default, handler: { [weak self] _ in
-            guard let self = self else { return }
-            self.openAppSettings()
-        }))
-
-        self.present(alert, animated: true, completion: nil)
-    }
-
-    private func openAppSettings() {
-        if let appSettings = URL(string: UIApplication.openSettingsURLString) {
-            UIApplication.shared.open(appSettings)
-        }
+    @objc func onRestorWalletTap() {
```

Comment:
`onRestoreWalletTap`

Reply:
Not sure about the general naming convention for tap events, if there's one let's follow it.